ts/Core/Navigator.ts

```diff
@@ -657,6 +658,18 @@ extend(defaultOptions, {
             threshold: null
         },
 
+        /**
+         * Enable or disable navigator sticking to right, while adding new
+         * points. If `undefined`, the default value is `true`.
```

Comment:
This is a bit unclear:

> If `undefined`, the default value is `true`.

I think the correct description is that "If `undefined`, the navigator keeps sticks to the axis maximum only if it was already at the maximum prior to adding points"